app/assets/stylesheets/application.sass

```diff
@@ -69,6 +69,9 @@ span.error
   color: $alert
   font-size: rem-calc(14)
 
+.br
+  height: rem-calc(20)
```

Comment:
Try keep it to margining and divisible by 8 (https://builttoadapt.io/intro-to-the-8-point-grid-system-d2573cde8632) if you wanna know why.